### PR TITLE
test settings change from non-default to default

### PR DESCRIPTION
**Pull request by pondrejk**
Extending the existing test to cover for 2157869, also removed some old skips from the test_settings file

Local result:

```
 pytest tests/foreman/ui/test_settings.py -k test_positive_update_login_page_footer_text
================================================ test session starts =================================================
collected 16 items / 15 deselected / 1 selected                                                                      

tests/foreman/ui/test_settings.py .                                                                            [100%]


============================== 1 passed, 15 deselected, 2 warnings in 126.77s (0:02:06) ==============================
```

### CODE REVIEW

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_settings.py -k test_positive_update_login_page_footer_text

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_settings.py -k test_positive_update_login_page_footer_text

File: tests/foreman/ui/test_settings.py [hunk 1]
```diff
@@ -210,30 +211,46 @@ def test_positive_update_login_page_footer_text(session, setting_update):
...
-            'username': settings.server.admin_username,
-            'password': settings.server.admin_password,
-        }
+
+        # change back to default (BZ#2157869)
+        session.login.login(login_details)
+        session.settings.update(f'name = {property_name}', default_value)
+        result = session.settings.read(f'name = {property_name}')
+        result = session.login.logout()
+        assert result["login_text"] == default_value
```
[the remark this answers is not in the document]
Comment (lhellebr) [reply tests/foreman/ui/test_settings.py:248]: Right, I missed line 240.

**Review by lhellebr — APPROVED** [review]

**Review by shweta83 — APPROVED** [review]
Ack

**Comment by pondrejk** [comment]
prt fails for all builds, not related to this pr, see the local result shared above

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_settings.py -k test_positive_update_login_page_footer_text

File: tests/foreman/ui/test_settings.py [hunk 2]
```diff
@@ -210,30 +209,45 @@ def test_positive_update_login_page_footer_text(session, setting_update):
         2. Click on general tab
         3. Input long length string into login page footer field
         4. Assert value from login page
-        5. Input empty string into the login page footer field
-        6. Assert empty value from login page
+        5. Input default value into the login page footer field
+        6. Assert default value from login page
+        Z. Input empty string into the login page footer field
```
Comment (rmynar) [inline tests/foreman/ui/test_settings.py:214]: Is that Z instead of 7 :-O

File: tests/foreman/ui/test_settings.py [hunk 3]
```diff
@@ -98,7 +98,6 @@ def test_positive_update_restrict_composite_view(session, setting_update, repo_s
                     session.contentview.delete(content_view_name)
 
 
-@pytest.mark.skip_if_open("BZ:1677282")
```
Comment (rmynar) [inline tests/foreman/ui/test_settings.py:101]: Shouldn't we keep the BZ numbers at least in docstrings (:BZ:)?

File: tests/foreman/ui/test_settings.py (hunk 2 above)
Comment (lhellebr) [reply tests/foreman/ui/test_settings.py:214]: That's to test if reviewers are attentive enough :D

Comment (pondrejk) [reply tests/foreman/ui/test_settings.py:214]: to my defense 
image

File: tests/foreman/ui/test_settings.py (hunk 3 above)
Comment (pondrejk) [reply tests/foreman/ui/test_settings.py:101]: ah ok, it is there, on line 112

**Review by vsedmik — APPROVED** [review]

**Review by rmynar — APPROVED** [review]

**Pull request merged**